dist/validation/schema.js

```diff
@@ -99,9 +99,9 @@ function onlyBlocksInCell(opts, change, context) {
 function noBlocksWithinCell(opts, change, context) {
   if (context.child.object === "block") {
     if (context.child.isVoid) {
-      return change.unwrapNodeByKey(context.child.key);
+      return change.removeNodeByKey(context.child.key, { normalize: false });
```

Comment:
Is this essentially removing the added paragraph tags if its child is void, to try making the tables more performant? I tried running the slate-tools branch locally and from my nightmare card created on the earlier version of slate, things look good, but I'll create some that have things inside of tables to help us test
https://qaapp.getguru.com/card/Tnnbjgac/Nightmare-Card-from-previous-slate-version (except I tested it locally)

Reply:
Yeah we noticed this was essentially backwards. The context should be the table cell, we are checking if the child is a block, if so then we are checking if it's void, like a divider, we don't want those so we remove them (also void won't have any text in it), while we want to keep any text, so we unwrap the paragraph block. We noticed it was backwards when doing some testing last night.

Reply:
Great, that makes sense. That's so interesting